actions/helm-deploy/action.yaml

```diff
@@ -0,0 +1,92 @@
+name: "Deploy helm chart"
+description: "Deploy a helm chart to the GKE cluster"
+inputs:
+  gke-service-account:
+    description: "GKE service account key for authentication"
+    required: true
+  gke-project:
+    description: "GKE project id for authentication"
+    required: true
+  gke-region:
+    description: "GKE region for authentication"
+    required: true
+  gke-cluster:
+    description: "GKE cluster for authentication"
+    required: true
```

Comment:
I think one thing I can improve here is allowing Kubernetes authentication mainly via kubeconfig file instead of restricting the action to GKE only

Reply:
Let's do that in a follow-up

Reply:
I created an issue: https://github.com/bakdata/ci-templates/issues/14